src/main/java/desktop/fragments/NavigationBar.java

```diff
@@ -0,0 +1,69 @@
+package desktop.fragments;
+
+import abstractclasses.fragment.AbstractFragment;
+import desktop.pages.*;
+import org.openqa.selenium.By;
+import org.openqa.selenium.WebDriver;
+import org.openqa.selenium.WebElement;
+import org.openqa.selenium.support.FindBy;
+
+public class NavigationBar extends AbstractFragment {
+
+    public NavigationBar(WebDriver driver) {
+        super(driver);
+    }
+
+    private static final By navigationBarElement = new By.ByXPath("//*[@class='user-nav']");
```

Comment:
Just a small hint - that you can use the same @FindBy annotation and WebElement for this navigation bar